PhysicsTools/NanoAOD/python/jets_cff.py

```diff
@@ -670,6 +675,28 @@
 for modifier in run2_miniAOD_80XLegacy, run2_nanoAOD_94X2016:
   modifier.toReplaceWith(jetSequence, _jetSequence_2016)
 
+#HF shower shape recomputation 
+#Only run if needed (i.e. if default MINIAOD info is missing or outdated because of new JECs...) 
+from RecoJets.JetProducers.hfJetShowerShape_cfi import hfJetShowerShape
+for modifier in run2_miniAOD_80XLegacy, run2_nanoAOD_94X2016, run2_nanoAOD_94XMiniAODv1, run2_nanoAOD_94XMiniAODv2, run2_nanoAOD_102Xv1:
```

Comment:
among the previous eras we should have 'run2_nanoAOD_106Xv1' 
those are the UL miniAOD out of the Summer19

Reply:
Indeed, I'll update that as well